pkg/model/dev.go

```diff
@@ -608,6 +612,20 @@ func setBuildDefaults(build *BuildInfo) {
 	}
 }
 
+func setBuildsManifestDefaults(builds ManifestBuild) {
+	for name, b := range builds {
+		if b == nil {
+			b = &BuildInfo{}
+		}
+		if b.Context == "" {
+			b.Context = fmt.Sprintf("./%s", name)
+		}
+		if _, err := url.ParseRequestURI(b.Context); err != nil && b.Dockerfile == "" {
+			b.Dockerfile = "Dockerfile"
```

Comment:
Why are we parsing `b.Context`? I think the default should be `b.Context` + `Dockerfile`
@jLopezbarb what's the default in stacks?